io.github.leeroymanea.synology-note-station.yaml

```diff
@@ -0,0 +1,42 @@
+# copied/inspired from https://github.com/flathub/com.synology.SynologyDrive/blob/master/com.synology.SynologyDrive.yaml
+id: io.github.leeroymanea.synology-note-station
```

Comment:
since this Synology's why is the app ID not their domain?

Reply:
Thank you for your advice. I will change it.

Reply:
Its done